velox/exec/tests/DriverTest.cpp

```diff
@@ -235,24 +235,31 @@ class DriverTest : public OperatorTestBase {
     std::lock_guard<std::mutex> l(wakeupMutex_);
     if (!wakeupInitialized_) {
       wakeupInitialized_ = true;
-      wakeupThread_ = std::thread([&]() {
+      wakeupThread_ = std::thread([this]() {
         int32_t counter = 0;
         for (;;) {
-          if (wakeupCancelled_) {
-            return;
+          {
+            std::lock_guard<std::mutex> l(wakeupMutex_);
```

Comment:
@oerling This shadows variable on line 235 and will also deadlock because of it.